hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/AutoAccountCreationSuite.java

```diff
@@ -1331,43 +1331,28 @@ final HapiSpec multipleAutoAccountCreations() {
-                    // Continue with more assertions given the normal case the preceding transfer succeeded
-                    if (actualStatus == SUCCESS) {
-                        allRunFor(
-                                spec,
-                                getTxnRecord("multipleAutoAccountCreates")
-                                        .hasNonStakingChildRecordCount(3)
-                                        .logged(),
-                                getAccountInfo(PAYER)
-                                        .has(accountWith()
-                                                .balance((INITIAL_BALANCE * ONE_HBAR) - 3 * ONE_HUNDRED_HBARS)),
-                                cryptoTransfer(
-                                                tinyBarsFromToWithAlias(PAYER, "alias4", 7 * ONE_HUNDRED_HBARS),
-                                                tinyBarsFromToWithAlias(PAYER, "alias5", 100))
-                                        .via("failedAutoCreate")
-                                        .hasKnownStatus(INSUFFICIENT_ACCOUNT_BALANCE),
-                                getTxnRecord("failedAutoCreate")
-                                        .hasNonStakingChildRecordCount(0)
-                                        .logged(),
-                                getAccountInfo(PAYER)
-                                        .has(accountWith()
-                                                .balance((INITIAL_BALANCE * ONE_HBAR) - 3 * ONE_HUNDRED_HBARS)));
-                    }
-                }));
+                                tinyBarsFromToWithAlias(PAYER, "alias1", ONE_HUNDRED_HBARS),
+                                tinyBarsFromToWithAlias(PAYER, ALIAS_2, ONE_HUNDRED_HBARS),
+                                tinyBarsFromToWithAlias(PAYER, "alias3", ONE_HUNDRED_HBARS)),
+                        "multipleAutoAccountCreates",
+                        getTxnRecord("multipleAutoAccountCreates")
+                                .hasNonStakingChildRecordCount(3)
+                                .logged(),
+                        getAccountInfo(PAYER)
+                                .has(accountWith().balance((INITIAL_BALANCE * ONE_HBAR) - 3 * ONE_HUNDRED_HBARS)),
+                        cryptoTransfer(
+                                        tinyBarsFromToWithAlias(PAYER, "alias4", 7 * ONE_HUNDRED_HBARS),
+                                        tinyBarsFromToWithAlias(PAYER, "alias5", 100))
+                                .via("failedAutoCreate")
+                                .hasKnownStatus(INSUFFICIENT_ACCOUNT_BALANCE),
+                        getTxnRecord("failedAutoCreate")
+                                .hasNonStakingChildRecordCount(0)
+                                .logged(),
+                        getAccountInfo(PAYER)
+                                .has(accountWith().balance((INITIAL_BALANCE * ONE_HBAR) - 3 * ONE_HUNDRED_HBARS))));
```

Comment:
The `allRunFor` seems to have disappeared, was that intentional?

Reply:
Yes, these operations are now executed directly by the  `assumingNoStakingChildRecordCausesMaxChildRecordsExceeded()` wrapper operation; it is equivalent.